backend/services/interfaces/authService.ts

```diff
@@ -11,15 +11,6 @@ interface IAuthService {
    */
   generateToken(email: string, password: string): Promise<AuthDTO>;
 
-  /**
```

Comment:
Removed since we also wanted to get rid of Google OAuth here since it won't be used and messes up some of the changes.